pkg/services/librarypanels/librarypanels_test.go

```diff
@@ -64,37 +72,43 @@ func TestGetLibraryPanel(t *testing.T) {
 		func(t *testing.T, sc scenarioContext) {
 			response := sc.service.getHandler(sc.reqContext)
 			require.Equal(t, 404, response.Status())
-		}, scenarioConfig{params: map[string]string{":panelId": "74"}})
```

Comment:
Why are you removing the panelId parameter instead of renaming it to uid? It seems to me you are introducing undefined behaviour into this scenario, by removing the parameter saying which panel to get.

Reply:
Just keeping it simple and that made it so I could remove the scenarioConfig completely. Might reintroduce it when needed. Also the `uid` in this case will be the empty string right?

Reply:
@hugohaggmark My argument is that it's still needed. Why should this scenario pass an empty string for uid? The scenario is getting a non-existent library panel, not omitting the uid parameter. Omitting it should be a validation error AFAICT.

Reply:
My opinion here is that the test case should pass a UID for the parameter, it will just not exist as in the code prior to the PR.

Reply:
Ok, I'll see what I can come up with :)

Reply:
Fixed in the latest commit